### PR TITLE
Remove superfluous Glean metrics

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Since we now have standardized page_load and click events, none of this custom code is required any more 🎉 

## Issue / Bugzilla link

N/A

## Testing

1. Open http://localhost:8000/en-US/
2. Set `window.Glean.setLogPings(true)` in the web console and refresh the page.

[ ] You should no longer see the following metrics included in the page_load event:

```
    "datetime": {
      "page.viewed": "2024-06-17T10:13:29.392+01:00"
    },
    "string": {
      "page.path": "/",
      "page.locale": "en-US",
      "page.referrer": "",
      "page.http_status": "200"
    },
    "labeled_string": {
      "page.query_params": {
        "utm_source": "",
        "utm_campaign": "",
        "utm_medium": "",
        "utm_content": "",
        "entrypoint_experiment": "",
        "entrypoint_variation": "",
        "experiment": "",
        "variation": "",
        "v": "",
        "xv": ""
      }
    },
```

### CODE REVIEW

File: tests/unit/spec/glean/utils.js [hunk 1]
```diff
@@ -12,134 +12,32 @@
...
-            expect(Utils.getUrl(url1)).toEqual(url1);
+            expect(Utils.filterURL(url1)).toEqual(url1);
 
             const url2 =
                 'https://www.mozilla.org/en-US/firefox/new/?utm_source=test&utm_campaign=test';
-            expect(Utils.getUrl(url2)).toEqual(url2);
+            expect(Utils.filterURL(url2)).toEqual(url2);
         });
 
         it('should remove newsletter tokens from know URLs', function () {
```
Comment (janbrasna) [inline tests/unit/spec/glean/utils.js:25]:
```suggestion
        it('should remove newsletter tokens from known URLs', function () {
```

File: tests/unit/spec/glean/utils.js [hunk 2]
```diff
@@ -12,134 +12,32 @@
 import Utils from '../../../../media/js/glean/utils.es6';
 
 describe('utils.js', function () {
-    describe('getUrl', function () {
+    describe('filterURL', function () {
         it('should return the a complete URL including query parameters', function () {
```
Comment (janbrasna) [inline tests/unit/spec/glean/utils.js:16]:
```suggestion
        it('should retain original URLs including query parameters', function () {
```

**Review by janbrasna — COMMENTED** [review]
Tiny description fixes that could be included along with it:

**Review by reemhamz — APPROVED** [review]
r+ 🏓

**Pull request merged**